### DISCUSSION TITLE
Help needed on receiving KNX Messages

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#12086] [Q&A] [answered]
Hello
have running a wall switch with Tasmota KNX. Sending to a group address at the KNX TP works fine.
![grafik](https://user-images.githubusercontent.com/48217146/117925672-db3ff480-b2f7-11eb-854d-74928a865c19.png)

```
07:36:33.017 KNX: Output 2 = 0 sent to 1.3.4
07:36:33.022 MQT: stat/tasmota_AA8CB0/RESULT = {"POWER2":"OFF"}
07:36:33.025 MQT: stat/tasmota_AA8CB0/POWER2 = OFF
07:40:54.045 KNX: Output 2 = 1 sent to 1.3.4
07:40:54.049 MQT: stat/tasmota_AA8CB0/RESULT = {"POWER2":"ON"}
07:40:54.053 MQT: stat/tasmota_AA8CB0/POWER2 = ON
```

But the device isn't receiving telegrams / group addresses ?
Should it be possible to receive group addresses sent by an actor at the TP bus ?

I'm using KNXd with these options and a KNX IP interface with 192.168.1.18
`--eibaddr=1.1.231 --client-addrs=1.1.240:1 -b ipt:192.168.1.18 --Routing --Server`

Any idea ? Thanks!

**Comment by ascillato** [comment, marked as answer]
>Should it be possible to receive group addresses sent by an actor at the TP bus ?

Yes, it is possible. Seems that your binding in KNXd is not complete.

You can use a phone app like MULTICAST TESTER in order to check if the messages are being retransmitted to your Wi-Fi network.

**Reply by Noschvie** [reply]
The "missing link" is the used WIFI configuration based on 2 devices "Aruba AP 205" with virtual controller.
Currently I don't know why KNX multicast routing works from WLAN -> LAN but not from LAN back to devices connected to the WLAN.
Using a "FRITZ!WLAN Repeater 1750E" as AP, the connected Tasmota devices are sending and receiving KNX multicast messages.

**Reply by ascillato** [reply]
Repeaters are known to have a very poor performance when resending multicast messages.

Anyway. Can you see the outgoing and ingoing KNX messages with any external tool like MULTICAST TESTER in your phone?

**Reply by Noschvie** [reply]
> Repeaters are known to have a very poor performance when resending multicast messages.

"FRITZ!WLAN Repeater 1750E" is just the product name. For the tests above it is used as AP connected to the LAN over ethernet, not used as repeater.

**Reply by Noschvie** [reply]
> Can you see the outgoing and ingoing KNX messages with any external tool like MULTICAST TESTER in your phone?

... got this test not up and running (not supported for iOS).

**Reply by Noschvie** [reply]
Had a look into the Aruba documentation and found the "Broadcast settings" as part of the "WLAN configuration parameters".
![grafik](https://user-images.githubusercontent.com/48217146/118365033-5248e780-b59b-11eb-87e7-240213ad388c.png)
... these settings are working! Tasmota devices are able to send and to receive KNX multicast messages to / from the TP bus.
